docs/cli.md

```diff
@@ -247,99 +255,99 @@ The type of catalog to be created is specified by the `--provider` option. Diffe
 ##### Create a Hive catalog
 
 ```bash
-gcli catalog create --metalake metalake_demo --name hive --provider hive --properties metastore.uris=thrift://hive-host:9083
+gcli catalog create --name hive --provider hive --properties metastore.uris=thrift://hive-host:9083
 ```
 
 ##### Create an Iceberg catalog
 
 ```bash
-gcli catalog create --metalake metalake_demo -name iceberg --provider iceberg --properties uri=thrift://hive-host:9083,catalog-backend=hive,warehouse=hdfs://hdfs-host:9000/user/iceberg/warehouse
+gcli catalog create  -name iceberg --provider iceberg --properties uri=thrift://hive-host:9083,catalog-backend=hive,warehouse=hdfs://hdfs-host:9000/user/iceberg/warehouse
 ```
 
 ##### Create a MySQL catalog
 
 ```bash
-gcli catalog create --metalake metalake_demo -name mysql --provider mysql --properties jdbc-url=jdbc:mysql://mysql-host:3306?useSSL=false,jdbc-user=user,jdbc-password=password,jdbc-driver=com.mysql.cj.jdbc.Driver
+gcli catalog create  -name mysql --provider mysql --properties jdbc-url=jdbc:mysql://mysql-host:3306?useSSL=false,jdbc-user=user,jdbc-password=password,jdbc-driver=com.mysql.cj.jdbc.Driver
 ```
 
 ##### Create a Postgres catalog
 
 ```bash
-gcli catalog create --metalake metalake_demo -name postgres --provider postgres --properties jdbc-url=jdbc:postgresql://postgresql-host/mydb,jdbc-user=user,jdbc-password=password,jdbc-database=db,jdbc-driver=org.postgresql.Driver
+gcli catalog create  -name postgres --provider postgres --properties jdbc-url=jdbc:postgresql://postgresql-host/mydb,jdbc-user=user,jdbc-password=password,jdbc-database=db,jdbc-driver=org.postgresql.Driver
 ```
 
 ##### Create a Kafka catalog
 
 ```bash
-gcli catalog create --metalake metalake_demo -name kafka --provider kafka --properties bootstrap.servers=127.0.0.1:9092,127.0.0.2:9092
+gcli catalog create --name kafka --provider kafka --properties bootstrap.servers=127.0.0.1:9092,127.0.0.2:9092
 ```
 
 #### Delete a catalog
 
 ```bash
-gcli catalog delete --metalake metalake_demo --name hive
+gcli catalog delete --name hive
 ```
 
 #### Rename a catalog
 
 ```bash
-gcli catalog update --metalake metalake_demo --name catalog_mysql --rename mysql
+gcli catalog update --name catalog_mysql --rename mysql
```

Comment:
In the above command, this mysql catalog is named "mysql"; in the subsequent commands, its name is "catalog_mysql", so I guess here, you want to rename it from "mysql" to "catalog_mysql", instead of the current one. Is it?

Reply:
No the command takes a catalog called "catalog_mysql" and renames it "mysql"

Reply:
But the "catalog_mysql" catalog wasn't there, its name is "mysql" before this step. If user follow the examples, he/she would get error at this step.

Reply:
'catalog_mysql' exists by default in the playground, so the user would not get an error.

Reply:
The `MySQL` catalogue is an extra one that is created on top of the existing playground one.

Reply:
I also note this has nothing to do with this PR, if it is a new issue it would be best to raised it and one and deal with it in another PR I think.